README.md

```diff
@@ -24,7 +24,9 @@ The block form offers customization beyond that of the inline form and support f
   {{#tm.target~}}
     Long text to truncate with <em><strong>really</strong> important</em> formatting.
   {{~/tm.target}}
-  {{#tm.button}}{{if tm.isTruncated "more" "less"}}{{/tm.button}}
+  {{#if tm.neededTruncating}}
```

Comment:
Can we just use the same `isTruncated` value already yielded?

Reply:
No because the yielded `isTruncated` corresponds to `truncationState` which only tells whether the text is currently truncated or not. 

We need an extra piece of state for the case where text was already too short to ever get truncated. Imagine a 3-line paragraph but the component is set to truncate at 20 lines. In this case, using just `isTruncated`, the component will show a "more" button. We don't want to offer a button at all in this case.

Reply:
This makes total sense. See my comment above.